MovieApp/MovieApp/MovieApp/ViewModels/BaseViewModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using MovieApp.Models;
+using MovieApp.Services;
+using Prism.Mvvm;
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Xamarin.Forms;
+
+namespace MovieApp.ViewModels
+{
+    public class BaseViewModel : BindableBase
+    {
+        bool isBusy = false;
```

Comment:
This should be a private property right?
Please follow your code standards: private bool _isBusy